utility/src/test/java/org/oppia/android/util/utility/NetworkConnectionTestUtil.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.util.utility
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import org.robolectric.Shadows
+import org.robolectric.shadows.ShadowNetworkInfo
+
+/** Test utility to modify [ShadowNetworkInfo] during the test. */
+class NetworkConnectionTestUtil {
+
+  companion object {
+    /**
+     * Sets the [ShadowNetworkInfo] during the test.
+     *
+     * @param context: the context of the dummy network connection.
+     * @param status: the type of network.
+     * @param networkState: [status] of the network connection.
```

Comment:
```suggestion
     * @param context the context of the dummy network connection
     * @param status the type of network
     * @param networkState status of the network connection
```

Note changes for KDoc formatting corrections.

Reply:
Done.